tests/integration/ha/test_backups.py

```diff
@@ -321,16 +316,13 @@ async def test_small_deployment_build_and_deploy(
         ops_test.model.deploy(charm, num_units=3, series=series, config=CONFIG_OPTS),
     )
 
-    # Relate it to OpenSearch to set up TLS.
     await ops_test.model.integrate(APP_NAME, TLS_CERTIFICATES_APP_NAME)
     await ops_test.model.wait_for_idle(
         apps=[TLS_CERTIFICATES_APP_NAME, APP_NAME],
         status="active",
         timeout=1400,
         idle_period=IDLE_PERIOD,
     )
-    # Credentials not set yet, this will move the opensearch to blocked state
-    # Credentials are set per test scenario
```

Comment:
why is this and other comments removed, they can serve as good documentation? please review the deleted ones and keep the ones that describe a state or an interesting flow